### PR TITLE
Update of Enterprise FAQ

### PR DESCRIPTION
**Pull request by pmanlukas**
What I did:
- update the existing links
- remove outdated information
- added a new section on actions and packages

### CODE REVIEW

**Comment by jonico** [comment]
I will look info your changes, thank you!

File: README.md [hunk 1]
```diff
@@ -150,5 +147,36 @@ Mit über 50 Millionen Software-Entwicklern, die GitHub monatlich nutzen, ist un
...
+
+[GitHub Package Registry](https://help.github.com/en/packages/publishing-and-managing-packages/about-github-packages) ist ein in GitHub integrierter Artefakt Storage. Dieser kann aktuell [NPM, Nugget, Ruby Gem, Maven, Gradle und Docker Images](https://help.github.com/en/packages/publishing-and-managing-packages/about-github-packages#supported-clients-and-formats) speichern. Die Package Registry kann sowohl für öffentliche als auch private Projekte verwendet werden und auch direkt aus GitHub Actions als Artefakt Storage verwendet werden. Auch kann Sie in andere Anwendungen eingebunden werden. Vom Funktionsumfang ist die Package Registry aber aktuell nicht mit einem Produkt wie Artifactory vergleichbar.
+
+* Sind GitHub Actions/ GitHub Package Registry für GitHub Enterprise verfügbar?
+
+Aktuell sind beide Produkte für GitHub Enterprise Cloud und Github.com verfügbar. Wir planen später in 2020 die Integration in GitHub Enterprise Server.
+
+* Kann ich Jenkins Files mit GitHub Actions wiederverwenden?
```
Comment (jonico) [inline README.md]:
```suggestion
* Kann ich Jenkinsfiles mit GitHub Actions wiederverwenden?
```

File: README.md [hunk 2]
```diff
@@ -150,5 +147,36 @@ Mit über 50 Millionen Software-Entwicklern, die GitHub monatlich nutzen, ist un
...
... GitHub Actions nutzt hierfür Pipelines, welche durch eine [YAML File](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions) definiert werden. Diese Pipelines bestehen aus sogenannten Actions, welche die einzelnen Schritte der Pipeline darstellen. Eine Action basiert entweder auf JavaScript Code oder einer Dockerfile und liegt in einem eigenen Repository auf GitHub. Es gibt nicht nur Actions von GitHub, sondern auch ein [wachsenden Ökosystem aus Actions](https://github.com/marketplace?type=actions) von anderen Anbietern (etwa Cloud Platformen wie AWS oder Azure oder auch Anbietern wie Atlassian). Eine Actions Pipeline kann auf mehr als [40 Events von GitHub](https://help.github.com/en/actions/reference/events-that-trigger-workflows) reagieren (wie einem Push oder Pull Request) und kann auch extern durch die API getriggered werden. Aus diesen Gründen eignet sich GitHub Actions auch für weitere Automatisierungsvorhaben auf GitHub (etwa die Moderation von Issues oder für Triage Prozesse). Auch kann man [eigene Actions erstellen](https://help.github.com/en/actions/building-actions) und diese wiederum innerhalb der Firma oder mit anderen GitHub Nutzern teilen.
```
Comment (jonico) [inline README.md]:
```suggestion
[GitHub Actions](https://help.github.com/en/actions/getting-started-with-github-actions/about-github-actions) ist GitHubs Platform um CI/CD Aufgaben, wie das Bauen von Code oder das automatisierte Testen von Komponenten, auf GitHub abzubilden. GitHub Actions nutzt hierfür Pipelines, welche durch ein [YAML-File](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions) definiert werden. Diese Pipelines bestehen aus sogenannten Actions, welche die einzelnen Schritte der Pipeline darstellen. Eine Action basiert entweder auf JavaScript Code oder einer Dockerfile und liegt in einem eigenen Repository auf GitHub. Es gibt nicht nur Actions von GitHub, sondern auch ein [wachsenden Ökosystem aus Actions](https://github.com/marketplace?type=actions) von anderen Anbietern (etwa Cloud-Platformen wie AWS oder Azure oder Dev-Tool-Herstellern wie Atlassian). Eine Actions Pipeline kann auf mehr als [40 Events von GitHub](https://help.github.com/en/actions/reference/events-that-trigger-workflows) reagieren (wie einem Push oder Pull Request) und kann auch extern durch die API getriggered werden. Aus diesen Gründen eignet sich GitHub Actions auch für weitere Automatisierungsvorhaben auf GitHub (etwa die Moderation von Issues oder für Triage Prozesse). Auch kann man [eigene Actions erstellen](https://help.github.com/en/actions/building-actions) und diese wiederum innerhalb der Firma oder mit anderen GitHub Nutzern teilen.
```

File: README.md [hunk 3]
```diff
@@ -150,5 +147,36 @@ Mit über 50 Millionen Software-Entwicklern, die GitHub monatlich nutzen, ist un
...
+
+* Sind GitHub Actions/ GitHub Package Registry für GitHub Enterprise verfügbar?
+
+Aktuell sind beide Produkte für GitHub Enterprise Cloud und Github.com verfügbar. Wir planen später in 2020 die Integration in GitHub Enterprise Server.
+
+* Kann ich Jenkins Files mit GitHub Actions wiederverwenden?
+
+Es gibt die Möglichkeit eine [Action](https://github.com/marketplace/actions/jenkinsfile-runner-prepackaged) zu nutzen, welche die Jenkinsfile erkennt und diese mittels eines [Jenkins Single Shot Master](https://github.com/SAP/devops-docker-cx-server/tree/master/jenkinsfile-runner-github-action) innerhalb von GitHub Actions als Docker Container ausführt. Die Jenkinsfile wird dabei innerhalb von Actions ausgeführt. Logs als auch Artefakte können hierbei von Actions abgegriffen und weiterverwendet werden. Dieser Ansatz ermöglicht die Wiederverwendung des bestehenden Jenkins Wissen. Unser Kunde SAP nutzt dies bereits für sein [Cloud SDK](https://blogs.sap.com/2019/09/13/running-sap-cloud-sdk-pipeline-on-github-actions/).
```
Comment (jonico) [inline README.md]:
```suggestion
Es gibt die Möglichkeit eine [Action](https://github.com/marketplace/actions/jenkinsfile-runner-prepackaged) zu nutzen, welche ```Jenkinsfiles``` erkennt und diese mittels eines [Jenkins Single Shot Master](https://github.com/SAP/devops-docker-cx-server/tree/master/jenkinsfile-runner-github-action) innerhalb von GitHub Actions als Docker Container ausführt. Die Jenkins-Pipeline wird dabei innerhalb von Actions ausgeführt. Logs als auch Artefakte können hierbei von Actions abgegriffen und weiterverwendet werden. Dieser Ansatz ermöglicht die Wiederverwendung des bestehenden Jenkins Wissen. SAP nutzt dies bereits für sein [Cloud SDK](https://blogs.sap.com/2019/09/13/running-sap-cloud-sdk-pipeline-on-github-actions/).
```

**Review by jonico — APPROVED** [review]

**Comment by jonico** [comment]
@pmanlukas: thanks for your ✨ contributions, see my suggestions inline, feel free to merge afterwards

**Comment by pmanlukas** [comment]
Thanks @jonico sadly I can only comment or close but not merge :(

**Pull request merged**